jfrog_platform_self-hosted/manifest.json

```diff
@@ -0,0 +1,81 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b2748652-b976-461c-91dd-5abd4467f361",
+  "app_id": "jfrog-platform",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View and analyze JFrog Artifactory and Xray Logs, Violations and Metrics",
+    "title": "JFrog Platform (Self-hosted)",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard",
+        "image_url": "images/jfrog_artifactory_logs.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Logo",
+        "image_url": "images/JFrog+Logo.png"
+      },
```

Comment:
```suggestion
```
The logo does not need to be added here, it will automatically be added to your tile.